json_object.c

```diff
@@ -961,7 +1049,12 @@ double json_object_get_double(const struct json_object *jso)
   case json_type_double:
     return jso->o.c_double;
   case json_type_int:
-    return jso->o.c_int64;
+    switch(jso->o.c_int.cint_type) {
+		case json_object_int_type_int64:
```

Comment:
something went wonky with the whitespace here.

Reply:
ok, modified.